src/net.cpp

```diff
@@ -613,7 +613,9 @@ void CNode::copyStats(CNodeStats &stats)
     X(fInbound);
     X(nStartingHeight);
     X(nSendBytes);
+    X(mSendBytesPerCmd);
```

Comment:
I don't care about following the original variable naming style, but if you do, the prefix for maps is map* not m*.